tensorflow_addons/optimizers/average_wrapper.py

```diff
@@ -103,14 +104,18 @@ def assign_average_vars(self, var_list):
         model.save('model.h5')
         ```
         """
-        assign_op = tf.group(
-            [
-                var.assign(self.get_slot(var, "average"), use_locking=self._use_locking)
-                for var in var_list
-                if var.trainable
-            ]
-        )
-        return assign_op
+        assign_ops = []
+        try:
+            for var in var_list:
+                assign_ops.append(
+                    var.assign(
```

Comment:
Should try-catch each assign op or all assign ops?

Reply:
Ops, it's my fault. Let me correct it real quick.

Reply:
Fixed. Thanks for pointing out!